### ISSUE TITLE
datalink::interfaces() fails to find wifi interfaces?

### ISSUE THREAD

**Issue by nikita-skobov** [libpnet/libpnet#374]
Is libpnet only meant to find virtual/ethernet interfaces, or should it be capable of finding wifi adapters as well?

On Windows 10 I can run: 
```wmic nic get guid, Name```

And I get the following output:
```
GUID                                    Name
{D232ADBC-815A-4053-B2E2-C236E89797F1}  Realtek PCIe GBE Family Controller
{CD790E52-B392-47CE-9DE4-7D6A9CBA1893}  Npcap Loopback Adapter
{183EBEF7-6997-4C89-9DC8-65F04A93F5A1}  VirtualBox Host-Only Ethernet Adapter
{EB0735F8-87D1-4129-8CAB-8C586797C4CB}  802.11n USB Wireless LAN Card #2 
```

When I use this example from the repo:
```rs
extern crate pnet;

use pnet::datalink;

fn main() {
  for interface in datalink::interfaces() {
    println!("{}", interface.name);
  }
}
```

It only gives me:
```
{183EBEF7-6997-4C89-9DC8-65F04A93F5A1}
{CD790E52-B392-47CE-9DE4-7D6A9CBA1893}
{D232ADBC-815A-4053-B2E2-C236E89797F1}
```

It does not show the wireless adapter that I use. I have tried several other wireless adapters and it still does not work.

I am on Windows 10, and using npcap version 0.996

**Comment by JuxhinDB** [comment]
I do not believe it should have issues. I had experienced something similar on Windows (missing interfaces, wifi or otherwise). However that was fixed by running as `SYSTEM` (admin). Could you try that out and let me know?

**Comment by nikita-skobov** [comment]
I tried opening CMD as administrator and running the rust program via the command line. same results as above. I also tried [this method](https://blogs.technet.microsoft.com/askds/2008/10/22/getting-a-cmd-prompt-as-system-in-windows-vista-and-windows-server-2008/) of opening CMD as SYSTEM and also got the same results

**Comment by JuxhinDB** [comment]
Do you have Wireshark installed on the same box? If so, are you able to pull the entire list of interfaces from there?

**Comment by nikita-skobov** [comment]
Yes I do, and I just checked: wireshark is in fact able to see all interfaces

**Comment by nikita-skobov** [comment]
I figured out my issue. It turns out I had [win10pcap](http://www.win10pcap.org) installed. win10pcap only works on ethernet devices. I figured this out when I uninstalled npcap and tried installing the old wpcap, which told me there already exists a wpcap on my computer. I searched for a bit and found that libpnet was using win10pcap, whereas wireshark was using npcap (which is why it was showing all of the interfaces correctly, but libpnet was not). I uninstalled both npcap and win10pcap and both libpnet and wireshark stopped showing any interfaces. I installed npcap on its own and now both wireshark and libpnet show all the right interfaces

😃
